### ISSUE TITLE
vitamin C level terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3192]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.   decreased vitamin C level
2.   decreased serum vitamin C level
3.   decreased liver vitamin C level
4.   increased vitamin C level
2.   increased serum vitamin C level
3.   increased liver vitamin C level

>2. Synonyms (e.g., absent spleen)

1.   reduced vitamin C level; reduced ascorbic acid level
2.   reduced serum vitamin C level; reduced serum ascorbic acid level
3.   reduced liver vitamin C level; reduced liver ascorbic acid level
4.   increased vitamin C level; increased ascorbic acid level
2.   increased serum vitamin C level; increased serum ascorbic acid level
3.   increased liver vitamin C level; increased liver ascorbic acid level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. any concentration below the normal range of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species
2.  any concentration below the normal range in serum of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species
3.  any concentration below the normal range in liver of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species
4. any concentration above the normal range of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species
5. any concentration above the normal range in serum of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species
6. any concentration above the normal range in liver of L-ascorbic acid (vitamin C), which is a common co-factor and has anti-oxidant activities in many species

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal vitamin C level (MP:0011229)
2.  decreased vitamin C level (new)
3.  decreased vitamin C level (new)
4.  abnormal vitamin C level (MP:0011229)
5.  increased vitamin C level (new)
6.  increased vitamin C level (new)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:gthayman

**Comment by anna-anagnostop** [comment]
abnormal vitamin C level MP:0011229 currently uses 'L-ascorbic acid' in the EQ. 

NOTE: abnormal vitamin C level MP:0011229 uses abnormal ascorbate level | abnormal ascorbic acid level | abnormal L-ascorbic acid as **EXACT** synonyms but CHEBI has different terms for ascorbate CHEBI:22651, ascorbic acid CHEBI:22652, and L-ascorbic acid CHEBI:29073

**Comment by anna-anagnostop** [comment]
Added new terms:

decreased vitamin C level MP:0031254
increased vitamin C level MP:0031255
abnormal circulating vitamin C level MP:0031256
decreased circulating vitamin C level MP:0031257
increased circulating vitamin C level MP:0031258
abnormal liver vitamin C level MP:0031259
decreased liver vitamin C level MP:0031260
increased liver vitamin C level MP:0031261
